crates/forge_analyzer/src/definitions.rs

```diff
@@ -2294,37 +2296,83 @@ impl Visit for FunctionCollector<'_> {
                 ..
             })) => {
                 if let Expr::Ident(ident) = &**expr {
-                    let ident = ident.to_id();
-                    let Some(def) = self.res.sym_to_id(ident, self.module) else {
-                        return;
-                    };
-                    if matches!(self.res.is_imported_from(def, "@forge/ui"), Some(ImportKind::Named(imp)) if *imp == *"render")
+                    let callee_ident = ident.to_id();
+                    if callee_ident.0 == "require"
+                        && self.res.sym_to_id(callee_ident, self.module).is_none()
+                        && args.len() == 1
                     {
-                        let owner =
-                            self.res
-                                .get_or_overwrite_sym(id, self.module, DefKind::Function(()));
-                        let Some(ExprOrSpread { expr, .. }) = &args.first() else {
-                            return;
-                        };
-                        let old_parent = self.parent.replace(owner);
-                        let mut analyzer = FunctionAnalyzer {
-                            res: self.res,
-                            module: self.module,
-                            current_def: owner,
-                            assigning_to: None,
-                            secret_packages: self.secret_packages.clone(),
-                            body: Body::with_owner(owner),
-                            block: BasicBlockId::default(),
-                            operand_stack: vec![],
-                            in_lhs: false,
+                        match args.get(0) {
+                            Some(ExprOrSpread { spread, expr: lit }) => {
+                                if let Expr::Lit(Lit::Str(Str { value, .. })) = &**lit {
+                                    let package_to_check = self
+                                        .secret_packages
+                                        .iter()
+                                        .find(|package| package.package_name == &**value);
+
```

Comment:
Can you add a `TODO` to handle local module imports?

Reply:
Done